src-ui/src/app/services/open-documents.service.ts

```diff
@@ -2,6 +2,9 @@ import { Injectable } from '@angular/core';
 import { PaperlessDocument } from '../data/paperless-document';
 import { OPEN_DOCUMENT_SERVICE } from '../data/storage-keys';
 import { DocumentService } from './rest/document.service';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
import { NgbModal } from "@ng-bootstrap/ng-bootstrap";
```